docker/dev/docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: "3.7"
+services:
+    opentogethertube-prod:
+        image: opentogethertube_dev
+        container_name: opentogethertube_dev
+        build:
+            context: ../../
+            dockerfile: docker/dev/Dockerfile
+        environment:
+            - YOUTUBE_API_KEY=API_KEY_GOES_HERE # optional put it in env/development.env
+            - REDIS_PASSWORD=""
+            - REDIS_DB=""
```

Comment:
IMO, we should just share the `env` folder to the container. I don't think it's a good idea to have people's configs contaminate their git working tree.